practice/src/scripts/constants/messages.ts

```diff
@@ -0,0 +1,6 @@
+const handleError = (functionName: string, error: Error): void => {
```

Comment:
what is purpose this function?

Reply:
This function is used to notify errors of each feature